wagtail_bucketav/views.py

```diff
@@ -11,12 +11,12 @@
 
 
 class BucketAVWebhookView(SNSEndpoint):
-    def handle_message(self, message, payload):
-        message = json.loads(message)  # Actually parse the JSON
+    def handle_message(self, message: str, payload) -> None:
+        message_dict = json.loads(message)  # Actually parse the JSON
```

Comment:
let's avoid re-using the same `message` variable as a string and then as a dict?